elide-spring/elide-spring-boot-autoconfigure/src/main/java/com/yahoo/elide/spring/dynamic/compile/ElideDynamicPersistenceUnit.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2020, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+package com.yahoo.elide.spring.dynamic.compile;
+
+import lombok.Data;
+
+import java.net.URL;
+import java.util.List;
+import java.util.Properties;
+
+import javax.persistence.SharedCacheMode;
+import javax.persistence.ValidationMode;
+import javax.persistence.spi.ClassTransformer;
+import javax.persistence.spi.PersistenceUnitInfo;
+import javax.persistence.spi.PersistenceUnitTransactionType;
+import javax.sql.DataSource;
+
+/**
+ * Persistent Unit implementation for Dynamic Configuration.
+ */
+@Data
+public class ElideDynamicPersistenceUnit implements PersistenceUnitInfo {
```

Comment:
This is a copy of another class in Elide.  Let's move it to a common place (probably JPA data store) - and have both pull from it.